package.json

```diff
@@ -45,7 +45,7 @@
     "shx": "^0.2.2",
     "through2-parallel": "^0.1.3",
     "yargs": "^8.0.2",
-    "@microsoft.azure/autorest.testserver": "1.9.2",
+    "@microsoft.azure/autorest.testserver": "2.3.9",
```

Comment:
@amarzavery FYI, bumped targeted test server version, remaining files are just from the regen (except for the package.json that I also commented on 😉 )

Your unit tests seem to fail right now (independent of this, see https://github.com/Azure/autorest.nodejs/pull/13). If you fix them, you will get [this](https://github.com/Azure/autorest.java/pull/38#issuecomment-344767962) commented on a PR automatically 😉